gettsim/transfers/grunds_im_alter.py

```diff
@@ -0,0 +1,445 @@
+        See :func:`grunds_im_alter_eink_m_hh`.
+
+    Returns
+    -------
+
+    """
+    out = (
+        regelbedarf_m_hh
+        + _grunds_im_alter_mehrbedarf_behinderung_m_hh
+        - grunds_im_alter_eink_m_hh
+        - unterhaltsvors_m_hh
+        - kindergeld_m_hh
+    )
+    return out.clip(lower=0)
+
+
+def grunds_im_alter_eink_m_hh(
+    grunds_im_alter_eink_m: FloatSeries, hh_id: IntSeries
+) -> FloatSeries:
+    """Aggregate income which is considered in the calculation of Grundsicherung im
+    Alter on household level.
+
+    Parameters
+    ----------
+    grunds_im_alter_eink_m
+        See :func:`grunds_im_alter_eink_m`.
+    hh_id
+        See basic input variable :ref:`hh_id <hh_id>`.
+
+    Returns
+    -------
+
+    """
+    return grunds_im_alter_eink_m.groupby(hh_id).sum()
+
+
+def grunds_im_alter_eink_m(
+    grunds_im_alter_erwerbseink_m: FloatSeries,
+    grunds_im_alter_priv_rente_m: FloatSeries,
+    grunds_im_alter_ges_rente_m: FloatSeries,
+    sonstig_eink_m: FloatSeries,
+    vermiet_eink_m: FloatSeries,
+    grunds_im_alter_kapitaleink_m: FloatSeries,
+    elterngeld_m: FloatSeries,
+    eink_st_tu: FloatSeries,
+    soli_st_tu: FloatSeries,
+    anz_erwachsene_tu: IntSeries,
+    sozialv_beitr_m: FloatSeries,
+    tu_id: IntSeries,
+    grunds_im_alter_params: dict,
+) -> FloatSeries:
+    """Calculate income which is considered in the calculation of Grundsicherung im
+    Alter.
+
+    Parameters
+    ----------
+    grunds_im_alter_erwerbseink_m
+        See :func:`grunds_im_alter_erwerbseink_m`.
+    grunds_im_alter_priv_rente_m
+        See :func:`grunds_im_alter_priv_rente_m`.
+    grunds_im_alter_ges_rente_m
+        See :func:`grunds_im_alter_ges_rente_m`.
+    sonstig_eink_m
+        See :func:`sonstig_eink_m`.
+    vermiet_eink_m
+        See :func:`vermiet_eink_m`.
+    grunds_im_alter_kapitaleink_m
+        See :func:`grunds_im_alter_kapitaleink_m`.
+    elterngeld_m
+        See :func:`elterngeld_m`.
+    eink_st_tu
+        See :func:`eink_st_tu`.
+    soli_st_tu
+        See :func:`soli_st_tu`.
+    anz_erwachsene_tu
+        See :func:`anz_erwachsene_tu`.
+    sozialv_beitr_m
+        See :func:`sozialv_beitr_m`.
+    tu_id
+        See basic input variable :ref:`tu_id <tu_id>`.
+    grunds_im_alter_params
+        See params documentation
+        :ref:`grunds_im_alter_params <grunds_im_alter_params>`.
+
+    Returns
+    -------
+
+    """
+
+    # Consider Elterngeld that is larger than 300
+    elterngeld_grunds_im_alter_m = (
+        elterngeld_m - grunds_im_alter_params["elterngeld_anr_frei"]
+    ).clip(lower=0)
+
+    # Income
+    total_income = (
+        grunds_im_alter_erwerbseink_m
+        + grunds_im_alter_ges_rente_m
+        + grunds_im_alter_priv_rente_m
+        + sonstig_eink_m
+        + vermiet_eink_m
+        + grunds_im_alter_kapitaleink_m
+        + elterngeld_grunds_im_alter_m
+    )
+
+    # subtract taxes and social security contributions
```

Comment:
```suggestion
    # subtract taxes and social insurance contributions
```
(search and replace for "social security" everywhere)